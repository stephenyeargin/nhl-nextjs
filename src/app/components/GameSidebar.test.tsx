import React from 'react';
import { render, screen, fireEvent, waitFor } from '@testing-library/react';
import GameSidebar from './GameSidebar';

// Lightweight child component mocks to isolate conditional rendering logic
jest.mock('./Scoreboard', () => ({ __esModule: true, default: () => <div data-testid="scoreboard">Scoreboard</div> }));
jest.mock('./TeamLogo', () => ({ __esModule: true, default: (p: any) => <span data-testid={`logo-${p.team || p.alt}`}>{p.team || p.alt}</span> }));
jest.mock('./StatComparisonRow', () => ({ __esModule: true, default: ({ stat }: any) => <div data-testid={`stat-${stat}`}>{stat}</div> }));
jest.mock('./FloatingVideoPlayer', () => ({ __esModule: true, default: ({ isVisible, label }: any) => isVisible ? <div data-testid="video-player">Video {label}</div> : null }));

// Mock formatter helpers that rely on date/time formatting for determinism
jest.mock('../utils/formatters', () => ({
  ...jest.requireActual('../utils/formatters'),
  formatSeriesStatus: () => 'Series Tied 1-1',
  formatLocalizedTime: () => '7:00 PM',
  formatPeriodLabel: () => 'P1',
  formatSeason: () => '2024-25'
}));

// Provide deterministic team data colors
jest.mock('../utils/teamData', () => ({
  getTeamDataByAbbreviation: () => ({ teamColor: '#123456', secondaryTeamColor: '#abcdef' })
}));

// Construct minimal but complete gameData structure hitting most conditional blocks
const buildGameData = () => ({
  homeTeam: { abbrev: 'HOM', logo: 'home.png', data: {}, },
  awayTeam: { abbrev: 'AWY', logo: 'away.png', data: {}, },
  game: {
    id: 99,
    gameState: 'FINAL',
    homeTeam: { abbrev: 'HOM' },
    awayTeam: { abbrev: 'AWY' },
    startTimeUTC: '2024-10-01T23:00:00Z',
    periodDescriptor: { number: 1 }
  },
  rightRail: {
    gameVideo: { threeMinRecap: '12345', condensedGame: '98765' },
    linescore: { dummy: true },
    shotsByPeriod: [ { away: 5, home: 7, periodDescriptor: { number: 1 } } ],
    teamSeasonStats: {
      contextLabel: 'REG', contextSeason: '20242025',
      awayTeam: { ppPctg: 10, ppPctgRank: 20, pkPctg: 80, pkPctgRank: 5, faceoffWinningPctg: 52, faceoffWinningPctgRank: 11, goalsForPerGamePlayed: 3.1, goalsForPerGamePlayedRank: 7, goalsAgainstPerGamePlayed: 2.7, goalsAgainstPerGamePlayedRank: 9 },
      homeTeam: { ppPctg: 15, ppPctgRank: 12, pkPctg: 82, pkPctgRank: 3, faceoffWinningPctg: 51, faceoffWinningPctgRank: 14, goalsForPerGamePlayed: 2.9, goalsForPerGamePlayedRank: 12, goalsAgainstPerGamePlayed: 2.5, goalsAgainstPerGamePlayedRank: 6 }
    },
    last10Record: {
      awayTeam: { record: '6-3-1', streakType: 'W', streak: 2, pastGameResults: [ { opponentAbbrev: 'ABC', gameResult: 'W' }, { opponentAbbrev: 'DEF', gameResult: 'L' } ] },
      homeTeam: { record: '5-4-1', streakType: 'L', streak: 1, pastGameResults: [ { opponentAbbrev: 'XYZ', gameResult: 'L' }, { opponentAbbrev: 'QRS', gameResult: 'W' } ] }
    },
    seasonSeries: [ {
      id: 1001,
      gameType: 2,
      gameState: 'FINAL',
      gameScheduleState: 'OK',
      awayTeam: { abbrev: 'AWY', logo: 'away.png', score: 2 },
      homeTeam: { abbrev: 'HOM', logo: 'home.png', score: 3 },
      periodDescriptor: { number: 3 },
      startTimeUTC: '2024-10-01T23:00:00Z',
      clock: { inIntermission: false, timeRemaining: '10:00' }
    } ],
    gameInfo: {
      referees: [ { default: 'Ref A' }, { default: 'Ref B' } ],
      linesmen: [ { default: 'Lines A' }, { default: 'Lines B' } ],
      awayTeam: { headCoach: { default: 'Coach Away' }, scratches: [] },
      homeTeam: { headCoach: { default: 'Coach Home' }, scratches: [] }
    },
    gameReports: { GAME_SUMMARY: 'https://example.com/report' }
  },
  story: {
    summary: {
      teamGameStats: [
        { category: 'sog', awayValue: 30, homeValue: 28 },
        { category: 'faceoffWinningPctg', awayValue: 45, homeValue: 55 },
        { category: 'powerPlayPctg', awayValue: 20, homeValue: 25 },
        { category: 'powerPlay', awayValue: 1, homeValue: 2 },
        { category: 'pim', awayValue: 6, homeValue: 8 },
        { category: 'hits', awayValue: 10, homeValue: 12 },
        { category: 'blockedShots', awayValue: 5, homeValue: 7 },
        { category: 'giveaways', awayValue: 3, homeValue: 1 },
        { category: 'takeaways', awayValue: 4, homeValue: 6 }
      ]
    }
  }
});

jest.mock('../contexts/GameContext', () => ({
  useGameContext: () => ({ gameData: buildGameData() })
}));

describe('GameSidebar (smoke)', () => {
  it('renders key sections and opens video player', async () => {
    render(<GameSidebar />);

    // Headings indicating most conditional blocks rendered
    expect(screen.getByText(/Game Stats/i)).toBeInTheDocument();
    expect(screen.getByText(/Season Stats/i)).toBeInTheDocument();
    expect(screen.getByText(/Last 10 Games/i)).toBeInTheDocument();
    expect(screen.getByText(/Game Info/i)).toBeInTheDocument();
    expect(screen.getByText(/Game Reports/i)).toBeInTheDocument();

    // Click recap button to trigger FloatingVideoPlayer visibility
    fireEvent.click(screen.getByText(/3:00 Recap/i));
    await waitFor(() => expect(screen.getByTestId('video-player')).toBeInTheDocument());
  });
});
