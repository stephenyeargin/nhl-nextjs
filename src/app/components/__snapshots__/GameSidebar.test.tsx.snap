// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GameSidebar (smoke) renders key sections and opens video player 1`] = `
<DocumentFragment>
  <div>
    <div
      class="flex justify-items-end gap-2 mb-3"
    >
      <button
        class="block p-1 rounded text-sm flex-1 text-center bg-blue-900 text-white font-bold hover:shadow hover:bg-blue-600"
      >
        <svg
          aria-hidden="true"
          class="svg-inline--fa fa-circle-play fa-fw "
          data-icon="circle-play"
          data-prefix="fas"
          focusable="false"
          role="img"
          viewBox="0 0 512 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M0 256a256 256 0 1 1 512 0A256 256 0 1 1 0 256zM188.3 147.1c-7.6 4.2-12.3 12.3-12.3 20.9l0 176c0 8.7 4.7 16.7 12.3 20.9s16.8 4.1 24.3-.5l144-88c7.1-4.4 11.5-12.1 11.5-20.5s-4.4-16.1-11.5-20.5l-144-88c-7.4-4.5-16.7-4.7-24.3-.5z"
            fill="currentColor"
          />
        </svg>
         3:00 Recap
      </button>
      <button
        class="block p-1 rounded text-sm flex-1 text-center bg-blue-900 text-white font-bold hover:shadow hover:bg-blue-600"
      >
        <svg
          aria-hidden="true"
          class="svg-inline--fa fa-circle-play fa-fw "
          data-icon="circle-play"
          data-prefix="fas"
          focusable="false"
          role="img"
          viewBox="0 0 512 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M0 256a256 256 0 1 1 512 0A256 256 0 1 1 0 256zM188.3 147.1c-7.6 4.2-12.3 12.3-12.3 20.9l0 176c0 8.7 4.7 16.7 12.3 20.9s16.8 4.1 24.3-.5l144-88c7.1-4.4 11.5-12.1 11.5-20.5s-4.4-16.1-11.5-20.5l-144-88c-7.4-4.5-16.7-4.7-24.3-.5z"
            fill="currentColor"
          />
        </svg>
         Condensed Game
      </button>
    </div>
    <div
      class="mb-5"
    >
      <div
        class="flex text-center items-center"
      >
        <div
          class="w-1/4 p-2 text-bold flex justify-center"
        >
          <span
            data-testid="logo-AWY"
          >
            AWY
          </span>
        </div>
        <div
          class="w-1/2 p-2 text-2xl font-bold"
        >
          Shots
        </div>
        <div
          class="w-1/4 p-2 text-bold flex justify-center"
        >
          <span
            data-testid="logo-HOM"
          >
            HOM
          </span>
        </div>
      </div>
      <div
        class="flex text-center bg-slate-500/10"
      >
        <div
          class="w-1/4 p-2"
        >
          5
        </div>
        <div
          class="w-1/2 p-3 text-xs"
        >
          P1
        </div>
        <div
          class="w-1/4 p-2"
        >
          7
        </div>
      </div>
    </div>
    <div
      class="mb-5"
    >
      <div>
        <div
          class="flex text-center items-center justify-between"
        >
          <div
            class="w-1/4 p-2 text-bold flex justify-center"
          >
            <span
              data-testid="logo-AWY"
            >
              AWY
            </span>
          </div>
          <div
            class="w-1/2 p-2 text-2xl font-bold"
          >
            Game Stats
          </div>
          <div
            class="w-1/4 p-2 text-bold flex justify-center"
          >
            <span
              data-testid="logo-HOM"
            >
              HOM
            </span>
          </div>
        </div>
        <div
          data-testid="stat-sog"
        >
          sog
        </div>
        <div
          data-testid="stat-faceoffWinningPctg"
        >
          faceoffWinningPctg
        </div>
        <div
          data-testid="stat-powerPlayPctg"
        >
          powerPlayPctg
        </div>
        <div
          data-testid="stat-pim"
        >
          pim
        </div>
        <div
          data-testid="stat-hits"
        >
          hits
        </div>
        <div
          data-testid="stat-blockedShots"
        >
          blockedShots
        </div>
        <div
          data-testid="stat-giveaways"
        >
          giveaways
        </div>
        <div
          data-testid="stat-takeaways"
        >
          takeaways
        </div>
      </div>
    </div>
    <div
      class="mb-5"
    >
      <div
        class="flex text-center items-center justify-between"
      >
        <div
          class="w-1/4 p-2 text-bold flex justify-center"
        >
          <span
            data-testid="logo-AWY"
          >
            AWY
          </span>
        </div>
        <div
          class="w-1/2 p-3"
        >
          <div
            class="text-2xl font-bold"
          >
            Season Stats
          </div>
          <div
            class="text-xs text-center text-gray-500"
          >
            2024-25 
          </div>
        </div>
        <div
          class="w-1/4 p-2 text-bold flex justify-center"
        >
          <span
            data-testid="logo-HOM"
          >
            HOM
          </span>
        </div>
      </div>
      <div
        data-testid="stat-ppPctg"
      >
        ppPctg
      </div>
      <div
        data-testid="stat-pkPctg"
      >
        pkPctg
      </div>
      <div
        data-testid="stat-faceoffWinningPctg"
      >
        faceoffWinningPctg
      </div>
      <div
        data-testid="stat-goalsForPerGamePlayed"
      >
        goalsForPerGamePlayed
      </div>
      <div
        data-testid="stat-goalsAgainstPerGamePlayed"
      >
        goalsAgainstPerGamePlayed
      </div>
    </div>
    <div
      class="mb-5"
    >
      <div
        class="flex text-center items-center justify-between"
      >
        <div
          class="w-1/4 p-2 text-bold flex justify-center"
        >
          <span
            data-testid="logo-AWY"
          >
            AWY
          </span>
        </div>
        <div
          class="w-1/2 p-2 text-2xl font-bold"
        >
          Last 10 Games
        </div>
        <div
          class="w-1/4 p-2 text-bold flex justify-center"
        >
          <span
            data-testid="logo-HOM"
          >
            HOM
          </span>
        </div>
      </div>
      <div
        class="flex text-center"
      >
        <div
          class="w-1/2"
        >
          6-3-1 (W2)
        </div>
        <div
          class="w-1/2"
        >
          5-4-1 (L1)
        </div>
      </div>
      <div
        class="flex text-center text-xs my-1 gap-1 bg-slate-500/10"
      >
        <div
          class="p-2 w-1/2 font-bold"
          style="border-width: 1pt; border-color: #123456;"
        >
          ABC (W)
        </div>
        <div
          class="p-2 w-1/2 opacity-50"
          style="border-width: 1pt; border-color: #123456;"
        >
          XYZ (L)
        </div>
      </div>
      <div
        class="flex text-center text-xs my-1 gap-1 "
      >
        <div
          class="p-2 w-1/2 opacity-50"
          style="border-width: 1pt; border-color: #123456;"
        >
          DEF (L)
        </div>
        <div
          class="p-2 w-1/2 font-bold"
          style="border-width: 1pt; border-color: #123456;"
        >
          QRS (W)
        </div>
      </div>
    </div>
    <div
      class="mb-5"
    >
      <div
        class="p-2 text-2xl font-bold text-center"
      >
        Season Series
      </div>
      <div
        class="text-center text-xs"
      >
        Series Tied 1-1
      </div>
      <div
        class="grid grid-cols-12 gap-3 py-4 items-center"
      >
        <a
          class="col-span-12 lg:col-span-6 p-1 mb-1 border rounded "
          href="/game/1001"
        >
          <div
            class="flex justify-between opacity-50"
          >
            <div
              class="flex items-center font-bold gap-1"
            >
              <span
                data-testid="logo-Logo"
              >
                Logo
              </span>
              AWY
            </div>
            <div
              class="text-lg font-bold"
            >
              2
            </div>
          </div>
          <div
            class="flex justify-between "
          >
            <div
              class="flex items-center font-bold gap-1"
            >
              <span
                data-testid="logo-Logo"
              >
                Logo
              </span>
              HOM
            </div>
            <div
              class="text-lg font-bold"
            >
              3
            </div>
          </div>
          <div
            class="flex justify-between"
          >
            <div>
              <span
                class="text-xs font-medium px-2 py-1 bg-slate-100 text-black rounded mr-1 uppercase"
              >
                Final
              </span>
            </div>
            <div>
              <span
                class="text-xs py-1"
              >
                Oct 1
              </span>
            </div>
          </div>
        </a>
      </div>
    </div>
    <div
      class="mb-5 text-xs"
    >
      <div
        class="p-2 text-2xl font-bold text-center"
      >
        Game Info
      </div>
      <div
        class="flex"
      >
        <div
          class="w-full p-2"
        >
          <div
            class="font-bold"
          >
            Officials
          </div>
          <div>
            Referees: 
            <span>
              Ref A
            </span>
            <span>
              , Ref B
            </span>
          </div>
          <div>
            Linesmen: 
            <span>
              Lines A
            </span>
            <span>
              , Lines B
            </span>
          </div>
        </div>
      </div>
      <div
        class="flex"
      >
        <div
          class="w-1/2 p-2"
        >
          <div
            class="my-2"
          >
            <div
              class="font-bold"
            >
              AWY Head Coach
            </div>
            Coach Away
          </div>
          <div
            class="my-2"
          >
            <div
              class="font-bold"
            >
              AWY Scratches
            </div>
            No players listed.
          </div>
        </div>
        <div
          class="w-1/2 p-2"
        >
          <div
            class="my-2"
          >
            <div
              class="font-bold"
            >
              HOM Head Coach
            </div>
            Coach Home
          </div>
          <div
            class="my-2"
          >
            <div
              class="font-bold"
            >
              HOM Scratches
            </div>
            No players listed.
          </div>
        </div>
      </div>
    </div>
    <div>
      <div
        class="p-2 text-2xl font-bold text-center"
      >
        Game Reports
      </div>
      <ul
        class="text-xs flex flex-wrap mt-2"
      >
        <li
          class="p-1 w-1/2 text-center"
        >
          <a
            class="font-bold underline"
            href="https://example.com/report"
          >
            GAME_SUMMARY
          </a>
        </li>
      </ul>
    </div>
  </div>
</DocumentFragment>
`;

exports[`GameSidebar (smoke) renders key sections and opens video player 2`] = `
<DocumentFragment>
  <div>
    <div
      class="flex justify-items-end gap-2 mb-3"
    >
      <button
        class="block p-1 rounded text-sm flex-1 text-center bg-blue-900 text-white font-bold hover:shadow hover:bg-blue-600"
      >
        <svg
          aria-hidden="true"
          class="svg-inline--fa fa-circle-play fa-fw "
          data-icon="circle-play"
          data-prefix="fas"
          focusable="false"
          role="img"
          viewBox="0 0 512 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M0 256a256 256 0 1 1 512 0A256 256 0 1 1 0 256zM188.3 147.1c-7.6 4.2-12.3 12.3-12.3 20.9l0 176c0 8.7 4.7 16.7 12.3 20.9s16.8 4.1 24.3-.5l144-88c7.1-4.4 11.5-12.1 11.5-20.5s-4.4-16.1-11.5-20.5l-144-88c-7.4-4.5-16.7-4.7-24.3-.5z"
            fill="currentColor"
          />
        </svg>
         3:00 Recap
      </button>
      <button
        class="block p-1 rounded text-sm flex-1 text-center bg-blue-900 text-white font-bold hover:shadow hover:bg-blue-600"
      >
        <svg
          aria-hidden="true"
          class="svg-inline--fa fa-circle-play fa-fw "
          data-icon="circle-play"
          data-prefix="fas"
          focusable="false"
          role="img"
          viewBox="0 0 512 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M0 256a256 256 0 1 1 512 0A256 256 0 1 1 0 256zM188.3 147.1c-7.6 4.2-12.3 12.3-12.3 20.9l0 176c0 8.7 4.7 16.7 12.3 20.9s16.8 4.1 24.3-.5l144-88c7.1-4.4 11.5-12.1 11.5-20.5s-4.4-16.1-11.5-20.5l-144-88c-7.4-4.5-16.7-4.7-24.3-.5z"
            fill="currentColor"
          />
        </svg>
         Condensed Game
      </button>
    </div>
    <div
      class="mb-5"
    >
      <div
        class="flex text-center items-center"
      >
        <div
          class="w-1/4 p-2 text-bold flex justify-center"
        >
          <span
            data-testid="logo-AWY"
          >
            AWY
          </span>
        </div>
        <div
          class="w-1/2 p-2 text-2xl font-bold"
        >
          Shots
        </div>
        <div
          class="w-1/4 p-2 text-bold flex justify-center"
        >
          <span
            data-testid="logo-HOM"
          >
            HOM
          </span>
        </div>
      </div>
      <div
        class="flex text-center bg-slate-500/10"
      >
        <div
          class="w-1/4 p-2"
        >
          5
        </div>
        <div
          class="w-1/2 p-3 text-xs"
        >
          P1
        </div>
        <div
          class="w-1/4 p-2"
        >
          7
        </div>
      </div>
    </div>
    <div
      class="mb-5"
    >
      <div>
        <div
          class="flex text-center items-center justify-between"
        >
          <div
            class="w-1/4 p-2 text-bold flex justify-center"
          >
            <span
              data-testid="logo-AWY"
            >
              AWY
            </span>
          </div>
          <div
            class="w-1/2 p-2 text-2xl font-bold"
          >
            Game Stats
          </div>
          <div
            class="w-1/4 p-2 text-bold flex justify-center"
          >
            <span
              data-testid="logo-HOM"
            >
              HOM
            </span>
          </div>
        </div>
        <div
          data-testid="stat-sog"
        >
          sog
        </div>
        <div
          data-testid="stat-faceoffWinningPctg"
        >
          faceoffWinningPctg
        </div>
        <div
          data-testid="stat-powerPlayPctg"
        >
          powerPlayPctg
        </div>
        <div
          data-testid="stat-pim"
        >
          pim
        </div>
        <div
          data-testid="stat-hits"
        >
          hits
        </div>
        <div
          data-testid="stat-blockedShots"
        >
          blockedShots
        </div>
        <div
          data-testid="stat-giveaways"
        >
          giveaways
        </div>
        <div
          data-testid="stat-takeaways"
        >
          takeaways
        </div>
      </div>
    </div>
    <div
      class="mb-5"
    >
      <div
        class="flex text-center items-center justify-between"
      >
        <div
          class="w-1/4 p-2 text-bold flex justify-center"
        >
          <span
            data-testid="logo-AWY"
          >
            AWY
          </span>
        </div>
        <div
          class="w-1/2 p-3"
        >
          <div
            class="text-2xl font-bold"
          >
            Season Stats
          </div>
          <div
            class="text-xs text-center text-gray-500"
          >
            2024-25 
          </div>
        </div>
        <div
          class="w-1/4 p-2 text-bold flex justify-center"
        >
          <span
            data-testid="logo-HOM"
          >
            HOM
          </span>
        </div>
      </div>
      <div
        data-testid="stat-ppPctg"
      >
        ppPctg
      </div>
      <div
        data-testid="stat-pkPctg"
      >
        pkPctg
      </div>
      <div
        data-testid="stat-faceoffWinningPctg"
      >
        faceoffWinningPctg
      </div>
      <div
        data-testid="stat-goalsForPerGamePlayed"
      >
        goalsForPerGamePlayed
      </div>
      <div
        data-testid="stat-goalsAgainstPerGamePlayed"
      >
        goalsAgainstPerGamePlayed
      </div>
    </div>
    <div
      class="mb-5"
    >
      <div
        class="flex text-center items-center justify-between"
      >
        <div
          class="w-1/4 p-2 text-bold flex justify-center"
        >
          <span
            data-testid="logo-AWY"
          >
            AWY
          </span>
        </div>
        <div
          class="w-1/2 p-2 text-2xl font-bold"
        >
          Last 10 Games
        </div>
        <div
          class="w-1/4 p-2 text-bold flex justify-center"
        >
          <span
            data-testid="logo-HOM"
          >
            HOM
          </span>
        </div>
      </div>
      <div
        class="flex text-center"
      >
        <div
          class="w-1/2"
        >
          6-3-1 (W2)
        </div>
        <div
          class="w-1/2"
        >
          5-4-1 (L1)
        </div>
      </div>
      <div
        class="flex text-center text-xs my-1 gap-1 bg-slate-500/10"
      >
        <div
          class="p-2 w-1/2 font-bold"
          style="border-width: 1pt; border-color: #123456;"
        >
          ABC (W)
        </div>
        <div
          class="p-2 w-1/2 opacity-50"
          style="border-width: 1pt; border-color: #123456;"
        >
          XYZ (L)
        </div>
      </div>
      <div
        class="flex text-center text-xs my-1 gap-1 "
      >
        <div
          class="p-2 w-1/2 opacity-50"
          style="border-width: 1pt; border-color: #123456;"
        >
          DEF (L)
        </div>
        <div
          class="p-2 w-1/2 font-bold"
          style="border-width: 1pt; border-color: #123456;"
        >
          QRS (W)
        </div>
      </div>
    </div>
    <div
      class="mb-5"
    >
      <div
        class="p-2 text-2xl font-bold text-center"
      >
        Season Series
      </div>
      <div
        class="text-center text-xs"
      >
        Series Tied 1-1
      </div>
      <div
        class="grid grid-cols-12 gap-3 py-4 items-center"
      >
        <a
          class="col-span-12 lg:col-span-6 p-1 mb-1 border rounded "
          href="/game/1001"
        >
          <div
            class="flex justify-between opacity-50"
          >
            <div
              class="flex items-center font-bold gap-1"
            >
              <span
                data-testid="logo-Logo"
              >
                Logo
              </span>
              AWY
            </div>
            <div
              class="text-lg font-bold"
            >
              2
            </div>
          </div>
          <div
            class="flex justify-between "
          >
            <div
              class="flex items-center font-bold gap-1"
            >
              <span
                data-testid="logo-Logo"
              >
                Logo
              </span>
              HOM
            </div>
            <div
              class="text-lg font-bold"
            >
              3
            </div>
          </div>
          <div
            class="flex justify-between"
          >
            <div>
              <span
                class="text-xs font-medium px-2 py-1 bg-slate-100 text-black rounded mr-1 uppercase"
              >
                Final
              </span>
            </div>
            <div>
              <span
                class="text-xs py-1"
              >
                Oct 1
              </span>
            </div>
          </div>
        </a>
      </div>
    </div>
    <div
      class="mb-5 text-xs"
    >
      <div
        class="p-2 text-2xl font-bold text-center"
      >
        Game Info
      </div>
      <div
        class="flex"
      >
        <div
          class="w-full p-2"
        >
          <div
            class="font-bold"
          >
            Officials
          </div>
          <div>
            Referees: 
            <span>
              Ref A
            </span>
            <span>
              , Ref B
            </span>
          </div>
          <div>
            Linesmen: 
            <span>
              Lines A
            </span>
            <span>
              , Lines B
            </span>
          </div>
        </div>
      </div>
      <div
        class="flex"
      >
        <div
          class="w-1/2 p-2"
        >
          <div
            class="my-2"
          >
            <div
              class="font-bold"
            >
              AWY Head Coach
            </div>
            Coach Away
          </div>
          <div
            class="my-2"
          >
            <div
              class="font-bold"
            >
              AWY Scratches
            </div>
            No players listed.
          </div>
        </div>
        <div
          class="w-1/2 p-2"
        >
          <div
            class="my-2"
          >
            <div
              class="font-bold"
            >
              HOM Head Coach
            </div>
            Coach Home
          </div>
          <div
            class="my-2"
          >
            <div
              class="font-bold"
            >
              HOM Scratches
            </div>
            No players listed.
          </div>
        </div>
      </div>
    </div>
    <div>
      <div
        class="p-2 text-2xl font-bold text-center"
      >
        Game Reports
      </div>
      <ul
        class="text-xs flex flex-wrap mt-2"
      >
        <li
          class="p-1 w-1/2 text-center"
        >
          <a
            class="font-bold underline"
            href="https://example.com/report"
          >
            GAME_SUMMARY
          </a>
        </li>
      </ul>
    </div>
    <div
      data-testid="video-player"
    >
      Video 3:00 Recap
    </div>
  </div>
</DocumentFragment>
`;
